woocommerce/woocommerce
# Add/wc addons tour base

### Description
We want to be sure that our customers know about In-App Marketplace. To introduce the latter to them we want to add a quick guided tour that shows in several steps what is the In-App Marketplace and how it can be used.

A related issue in WooCommerce.com repo: https://github.com/Automattic/woocommerce.com/issues/14749

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/woocommerce.com/issues/14749 (WooCommerce.com)






### How to test the changes in this Pull Request:


1. Checkout branch from this PR in your local environment
2. Build assets: `pnpm install && pnpm run build` (how to [get started](https://github.com/woocommerce/woocommerce#getting-started))
3. To run local dev environment: `cd plugins/woocommerce && npm run env:dev`
4. Your `http://localhost:8086/` should be available (assume that it's your base URL)
5. Open http://localhost:8086/wp-admin/admin.php?page=wc-addons&tutorial=true and make sure you see the first step:
<img width="635" alt="image" src="https://user-images.githubusercontent.com/115007291/197575979-8c631391-1657-43fa-8bea-fbe8bd524fe4.png">

6. Go to step 2
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/115007291/197576855-30b1e9c7-4700-4ce6-9618-36947b961fa9.png">

7. Then go to step 3
<img width="1230" alt="image" src="https://user-images.githubusercontent.com/115007291/197576906-2010eef2-dcc7-4f0f-b18e-54920c5a93b9.png">

8. And finally, go to step 4
<img width="1221" alt="image" src="https://user-images.githubusercontent.com/115007291/197576962-7022bdf7-1c40-42c3-9914-a67eb285b7cf.png">

9. Click the "Done button", it should finish the tour

10. Start from the beginning, you should be able to dismiss the tour by clicking on the ╳ ("Cross") icon in any step
11. Check that removing `tutorial=true` from the link prevents starting the tour



### Note
The WooCommerce Extensions page might show admin alerts that would pop up after the tour scrolled the window to show a pop-up dialog. Since this PR doesn't contain the fix for this issue, it will be fixed in a separate PR. H

### Other information:




### FOR PR REVIEWER ONLY:
